src/scripts/commander.js

```diff
@@ -53,14 +59,15 @@ program
     'The package name of the app to record data from (e.g., com.example.myapp)',
     ''
   )
-  .action((cmdObj) => {
+  .action(async (cmdObj) => {
+    console.log('------------reached');
```

Comment:
```suggestion
    console.log('------------reached');
```

Can we please add more meaningful log here?